src/Plugins/RestServer/Binder/UInt160BinderProvider.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2015-2025 The Neo Project.
+//
+// UInt160BinderProvider.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Binders;
+using System;
+
+namespace Neo.Plugins.RestServer.Binder
+{
+    internal class NeoBinderProvider : IModelBinderProvider
```

Comment:
The file name is different than the class name

Reply:
should be fixed